### ISSUE TITLE
System for adding axioms

### ISSUE THREAD

**Issue by io12** [Bram-Hub/aris#63]

**Comment by Bram28** [comment]
Yes, there is an axiom of induction, if you can quantify over formulas with free variables:

Weak Mathematical Induction:

forall P(x)  [(P(0) & forall x (P(x) -> P(s(x)))) -> forall x P(x)]

Strong Mathematical Induction:

forall P(x)  [forall x (forall y (y < x -> P(y)) -> P(x)) -> forall x P(x)]
